packages/http-server/src/serializers.ts

```diff
@@ -11,6 +11,13 @@ export default [
     },
     serializer: JSON.stringify,
   },
+  {
+    regex: {
```

Comment:
You do not need to create a new section, you can simply modify the one at line 9 and add `application/json` in the array